torch/csrc/jit/symbolic_script.cpp

```diff
@@ -1371,6 +1357,29 @@ const std::vector<std::string> functions = {
                 return grad_output, None
             return torch.remainder(self, other), backward
 
+        def AD_column_major(mat):
+            return mat.stride(0)==1 and mat.stride(1) == mat.size(0)
+
+        def AD_mm_backward_self(grad, mat2, mat1):
+            if AD_column_major(mat1):
```

Comment:
Should this error if mat1 is sparse like the C++ does?

Reply:
raising RuntimeError fails in interpreter, and this behavior change is beyond the scope of this PR. Sparse support is experimental in script (and a warning to this effect is printed when running with sparse tensors).